test/recipes/80-test_ssl_old.t

```diff
@@ -567,7 +567,7 @@ sub testssl {
 
             ok(run(test([@ssltest, "-bio_pair", "-tls1", "-cipher", "PSK", "-psk", "abc123"])),
                'test tls1 with PSK via BIO pair');
-	  }
+          }
```

Comment:
The indentation of the `SKIP: {` blocks and related closing braces is a bit wonky throughout the file.
A pass could be done to improve the indentation, but I am likely not the right person because I don't fully grasp the convention used for our Perl test recipes.

I can do such a pass in this file if requested, but I would need guidance from @mattcaswell